foundry/applications/dialog.d.ts

```diff
@@ -205,12 +205,12 @@ declare class Dialog extends Application {
       content: string;
       yes?: (html: JQuery | HTMLElement) => Y;
       no?: (html: JQuery | HTMLElement) => N;
-      render: (html: JQuery | HTMLElement) => void;
+      render?: (html: JQuery | HTMLElement) => void;
       defaultYes?: boolean;
       rejectClose?: R;
       options?: Partial<Dialog.Options>;
     },
-    old: Partial<Dialog.Options>
+    old?: Partial<Dialog.Options>
```

Comment:
this may not be optional in this overload because the parameter's existence is the way the function checks if tho e old, deprecated calling convention has been used.

Reply:
hmm.. not sure why, but TSC was giving me a problem with the `confirm` dialog. It wouldn't let me compile unless I made this argument optional. I've reverted the change locally and it seems to be allowing it now. Maybe this has been fixed since I rebased today.